public/app/features/org/team_details_ctrl.ts

```diff
@@ -1,15 +1,21 @@
 import coreModule from 'app/core/core_module';
+import config from 'app/core/config';
 
 export default class TeamDetailsCtrl {
   team: Team;
   teamMembers: User[] = [];
   navModel: any;
+  teamGroups: TeamGroup[] = [];
+  newGroupId: string;
+  enterprise: boolean;
 
   /** @ngInject **/
   constructor(private $scope, private backendSrv, private $routeParams, navModelSrv) {
     this.navModel = navModelSrv.getNav('cfg', 'teams', 0);
     this.userPicked = this.userPicked.bind(this);
     this.get = this.get.bind(this);
+    this.newGroupId = '';
+    this.enterprise = config.enterprise;
```

Comment:
long term I think feature toggles might be better, will make it easier to introduce more licence levels than oss & enterprise where depending on the license level different features are enabled/disabled.  But not needed right now, just a thought.

Reply:
Agree.